activities/app_configs.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse_lazy
+
+from activities.user_data import UserSocialAuthUserData
+from common.app_configs import BaseConnectionAppConfig
+
+
+class UserSocialAuthAppConfig(BaseConnectionAppConfig):
+    """
+    An AppConfig used for activities that connect via UserSocialAuth.
+    """
+
+    def get_user_data(self):
+        outer_self = self
+
+        @property
+        def get_user_data_inner(self):
```

Comment:
Trying to wrap my head around this wrapping... Why was it wanted/needed/useful? Also, does `user=self` mean a UserSocialAuthUserData object is created with this UserSocialAuthAppConfig object as the `user` parameter?